test/utils/setupTruefi2.ts

```diff
@@ -84,27 +84,25 @@ export const setupTruefi2 = async (owner: Wallet, provider: MockProvider, custom
   await lender.initialize(stkTru.address, poolFactory.address, customDeployed?.oneInch ? customDeployed.oneInch.address : AddressZero, creditOracle.address, rateAdjuster.address, borrowingMutex.address)
   await safu.initialize(loanFactory.address, liquidator.address, customDeployed?.oneInch ? customDeployed.oneInch.address : AddressZero)
   await poolFactory.initialize(implementationReference.address, lender.address, safu.address)
-  await rateAdjuster.initialize()
+  await rateAdjuster.initialize(poolFactory.address)
   await creditAgency.initialize(creditOracle.address, rateAdjuster.address, borrowingMutex.address, poolFactory.address)
   await standardBaseRateOracle.initialize(mockSpotOracle.address, standardToken.address, DAY)
   await feeBaseRateOracle.initialize(mockSpotOracle.address, feeToken.address, DAY)
 
   await poolFactory.allowToken(feeToken.address, true)
   await poolFactory.createPool(feeToken.address)
   const feePool = poolImplementation.attach(await poolFactory.pool(feeToken.address))
-  await poolFactory.supportPool(feePool.address)
   await feePool.setOracle(feeTokenOracle.address)
 
   await poolFactory.allowToken(standardToken.address, true)
   await poolFactory.createPool(standardToken.address)
   const standardPool = poolImplementation.attach(await poolFactory.pool(standardToken.address))
-  await poolFactory.supportPool(standardPool.address)
   await standardPool.setOracle(standardTokenOracle.address)
 
   await rateAdjuster.setBaseRateOracle(standardPool.address, standardBaseRateOracle.address)
   await rateAdjuster.setBaseRateOracle(feePool.address, feeBaseRateOracle.address)
-  await rateAdjuster.addPoolToTVL(standardPool.address)
-  await rateAdjuster.addPoolToTVL(feePool.address)
+  await poolFactory.supportPool(standardPool.address)
+  await poolFactory.supportPool(feePool.address)
```

Comment:
Let's delete these two lines and keep the `poolFactory.supportPool()` calls with poolFactory instead of rateAdjuster. This probably came from a git merge.